### PR TITLE
Update rendered Markdown styles

### PR DESCRIPTION
**Pull request by matt-west**
Fixes https://github.com/Automattic/dotcom-forge/issues/7670

## Proposed Changes

- Cleaned up styles for rendered markdown.
- Added left/right margins to messages.
- Updated background for user messages.

> Screenshot 2024-06-12 at 11 29 55
> Screenshot 2024-06-12 at 11 30 03

## Testing Instructions

- Open Studio using `STUDIO_AI=true npm start`
- Switch to the Assistant tab
- Enter a prompt to see example markdown styles:

```
Show me how all markdown elements would look when rendered
```

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: src/components/content-tab-assistant.tsx [hunk 1]
```diff
@@ -151,11 +151,17 @@ export const Message = ( { children, isUser, className }: MessageProps ) => {
 	};
 
 	return (
-		<div className={ cx( 'flex mt-4', isUser ? 'justify-end' : 'justify-start', className ) }>
+		<div
+			className={ cx(
+				'flex mt-4',
+				isUser ? 'justify-end md:ml-24' : 'justify-start md:mr-24',
```
Comment (matt-west) [inline src/components/content-tab-assistant.tsx:157]: Drive-by fix, to add the left/right margins to messages so they appear to the appropriate side of the chat history.

Screenshot 2024-06-12 at 11 33 29

**Review by jeroenpf — APPROVED** [review]
This works great and looks much better!

I noticed that images are not loaded if they are not on https://*.gravatar.com https://*.wp.com - not related to this PR but i guess we need to think about that at some point because when we switch to Odie, images will be used in the responses. See: https://mc.a8c.com/odie/odie-chat.php?chat_id=482889&page=2

edit: I also noticed tables aren't rendered yet and created an issue for this: https://github.com/Automattic/dotcom-forge/issues/7720

**Comment by matt-west** [comment]
@jeroenpf Interesting. Do you know if that's due to a limitation we've put in place?

**Comment by jeroenpf** [comment]
> @jeroenpf Interesting. Do you know if that's due to a limitation we've put in place?

It seems that we've added that here: https://github.com/Automattic/studio/blob/trunk/src/index.ts#L197 

@p-jackson might know more about this.

**Comment by matt-west** [comment]
Thanks! I’ve created an issue for fixing the images. https://github.com/Automattic/dotcom-forge/issues/7721
